disperser/dataapi/subgraph_client_test.go

```diff
@@ -527,23 +535,20 @@ func TestQueryOperatorQuorumEvent(t *testing.T) {
 	assert.Equal(t, 2, len(added1))
 	assert.Equal(t, "operator-1", added1[0].Operator)
 	assert.Equal(t, uint32(80), added1[0].BlockNumber)
-	assert.Equal(t, 2, len(added1[0].QuorumNumbers))
-	// Note: the quorumId is 48 not 01 is because the string "01" is in UTF-8
```

Comment:
previous assumption wasn't right; based on actual subgraph data, it's just a hex string starting with "0x".